packages/file-type-icons/src/FileTypeIconMap.ts

```diff
@@ -331,10 +331,10 @@ export const FileTypeIconMap: { [key: string]: { extensions?: string[] } } = {
   },
   multiple: {},
   one: {
-    extensions: ['one'], // this is a format for exported single - file notebook pages
+    extensions: ['note', 'one'], // This represents a single Fluid Note, or a partial exported section/page of a notebook.
```

Comment:
Looks like this line got too long
```suggestion
    // This represents a single Fluid Note, or a partial exported section/page of a notebook.
    extensions: ['note', 'one'],
```